apps/fabric/src/DotLottieExample.tsx

```diff
@@ -0,0 +1,26 @@
+import LottieView from 'lottie-react-native';
+import {View} from 'react-native';
+
+const dotLottie = require('./animations/animation_lkekfrcl.lottie');
+const jsonLottie = require('./animations/LottieLogo1.json');
+const remoteLottie = {
+  uri: 'https://raw.githubusercontent.com/lottie-react-native/lottie-react-native/master/apps/paper/src/animations/Watermelon.json',
+};
+
+export const DotLottieExample = () => {
+  return (
+    <LottieView
+      // source={dotLottie}
```

Comment:
Maybe make it a state to switch between the three of them so that people running the example can have a more interactive experience?

Reply:
Removed.